components/Header.js

```diff
@@ -26,8 +26,8 @@ export default class Header extends React.Component {
           <meta name="twitter:site" content="@openobservatory" />
           <meta name="twitter:creator" content="@openobservatory" />
 
-          <meta name='og:title' property='og:title' content='OONI Explorer - Open Data on Internet Censorship Worldwide' />
-          <meta property="og:description" content={description} />
+          <meta key="og:title" name='og:title' property='og:title' content='OONI Explorer - Open Data on Internet Censorship Worldwide' />
+          <meta key="og:description" property="og:description" content={description} />
```

Comment:
I think there is inconsistency (even before this PR) in how we use `property` and `name` attributes. But I have opened another issue (#450) to check that. We can retain this as it is for now.

Reply:
Ok, I'll look that up.